### DISCUSSION TITLE
Pinia not showing up on devtools and my issue was deleted

### DISCUSSION THREAD

**Discussion by slnw** [vuejs/pinia#2613] [Help and Questions] [answered]
Hi, I've setup pinia on Nuxt 3 and it doesn't show on my devtools I posed an issue about it but it was deleted. Why was that and how can I get it checked?

Deleted issue link - https://github.com/vuejs/pinia/issues/2610

Thanks,

**Comment by posva** [comment, marked as answer]
Nuxt Devtools are in the Nuxt repository; it's unrelated to pinia or the code in this repository. AFAIK they were still adding the feature: https://github.com/nuxt/devtools/commit/56be5a73585b9fd0f3169915bc2169dbe96379e7

Non-conforming issues are deleted to avoid affecting search results.
